ram/wram0.asm

```diff
@@ -258,12 +265,6 @@ wSelectorCur:: db
 ; song editor
 wChannelSelector:: db
 wAdjustingTempo:: db
-; audio engine input
-wChannelSelectorSwitches:: ds 4
```

Comment:
Now that this is no longer in the range of `wMusicPlayerWRAMEnd - wMusicPlayerWRAM`, does the "Initialize wMusicPlayerWRAM" need updating?

Reply:
This does mean no easy way To clear them, will make b from the editor do so

Reply:
Thanks!